src/main/java/com/hpe/adm/octane/ideplugins/services/EntityService.java

```diff
@@ -67,6 +67,14 @@
     }
 
     public Collection<EntityModel> findEntities(Entity entity, Query.QueryBuilder query, Set<String> fields, Map<String, Set<String>> expand) {
+        return findEntities(entity, query, fields, expand, null, null);
```

Comment:
@atothhpe Small personal preference: I would instead call findEntities(entity, query, fields, expand, null, null, null, null); here instead of the "intermediary" method, so that we don't have wrappers over other wrappers.
It's up to you